zubhub_frontend/zubhub/src/views/projects/Projects.jsx

```diff
@@ -206,7 +208,9 @@ function Projects(props) {
             ) : null}
             {next_page ? (
               <CustomButton
-                className={classes.floatRight}
+                // className={classes.floatRight}
+
+                className={`${classes.floatRight} ${classes.buttonGroupStyleAlternative}`}
```

Comment:
for consistency sake, you should use `clsx` when you want to add two classes to an element. search for `clsx` on your code editor to get an idea of what I mean